WordPress/gutenberg
# Update pagination icons, add new query title and post terms icons

## Description

Fixes #37541. 

This adds a couple of new icons:

- Query Title
- Post Terms

And updates from the latest in Figma a few existing ones:

- Pagination
- Pagination Next
- Pagination Previous
- Pagination Numbers

37541 also details "Post Navigation Link", but I wasn't sure exaclty what that was. Can we just use "Post Navigation" for that? 

Note also that the "numbers" icon appears to be unused, but is still in the package.

Pagination before and after:
<img width="178" alt="pagination before" src="https://user-images.githubusercontent.com/1204802/152526959-fa5373a1-dd4a-4d11-8fa7-13899afe28b6.png">

<img width="187" alt="pagination after" src="https://user-images.githubusercontent.com/1204802/152526966-92364a49-5d08-4f67-914e-bde305189002.png">

Next/prev before and after:
<img width="233" alt="nextprevbefore" src="https://user-images.githubusercontent.com/1204802/152527033-919a3521-4edd-4153-93f4-f47835b6d11a.png">

<img width="251" alt="nextprevafter" src="https://user-images.githubusercontent.com/1204802/152527038-8052adf7-cea6-4aef-aaa4-bab974d18be0.png">

Numbers after:

<img width="841" alt="numbersafter" src="https://user-images.githubusercontent.com/1204802/152527061-01854526-ed38-4315-b1a1-8ee861ab1fbb.png">

New Query icon shown in context of Post title in Figma:

<img width="722" alt="post title after" src="https://user-images.githubusercontent.com/1204802/152527098-d9e61f44-07a5-4843-a869-2f5919fe1c48.png">

Post terms in Figma:

<img width="841" alt="post terms after" src="https://user-images.githubusercontent.com/1204802/152527135-0af4721c-4b60-48f6-be42-ade9a5bea043.png">

Query title and post terms shown in "all blocks" context:

<img width="278" alt="Screenshot 2022-02-04 at 13 07 33" src="https://user-images.githubusercontent.com/1204802/152527182-f72d1713-a492-4a3c-a7f7-ea618fca6827.png">

<img width="267" alt="Screenshot 2022-02-04 at 13 08 15" src="https://user-images.githubusercontent.com/1204802/152527187-de90eb45-b841-43c3-a313-adab8b0e66f4.png">



## Testing Instructions

Open global styles, select "All blocks" and observe new icons for pagination, pagination next, pagination previous, query title, post terms.

## Checklist:
